mds_asssignment_1/guestbook/forms.py

```diff
@@ -0,0 +1,7 @@
+from django import forms
+
+class EntryForm(forms.Form):
+    fname = forms.CharField(max_length=50, label='First name')
```

Comment:
Nothing particularly wrong with separating first and last names, but I've moved away from it as much as possible in my own code. Here's w3's giant write up about why it's not always a good idea:
http://www.w3.org/International/questions/qa-personal-names.en.php?changelang=en